pom.xml

```diff
@@ -102,6 +102,7 @@
     <dependency>
       <groupId>org.jenkins-ci.plugins</groupId>
       <artifactId>git-client</artifactId>
+      <version>4.7.0</version>
```

Comment:
This can be removed since it is the version provided by the plugin bill of materials on the master branch.

Reply:
Removal suggestion
```suggestion
```